csharp/sample/Microsoft.ML.OnnxTraining.MNISTSample/BinaryFile.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) 2018-2020 SignalPop LLC, and contributors. All rights reserved.
+// License: Apache 2.0
+// License: https://github.com/MyCaffe/MyCaffe/blob/master/LICENSE
+// Modified from Original Source: https://github.com/MyCaffe/MyCaffe/blob/master/MyCaffe.data/MnistDataLoader.cs
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+/// <summary>
+/// The MyCaffe.data namespace contains dataset creators used to create common testing datasets such as MNIST, CIFAR-10 and VOC0712.
+/// </summary>
+namespace MyCaffe.data
+{
+    /// <summary>
+    /// The BinaryFile class is used to manage binary files used by the MNIST dataset creator.
+    /// </summary>
+    class BinaryFile : IDisposable
+    {
+        FileStream m_file;
+        BinaryReader m_reader;
```

Comment:
What is the access level for this?

Reply:
public.

It is used on line 189 of
https://github.com/MyCaffe/onnxruntime/blob/WinTrainingCSharp/csharp/sample/Microsoft.ML.OnnxTraining.MNISTSample/MnistDataLoaderLite.cs

Reply:
Updated